troubleshooting/_posts/2016-12-12-my-build-is-failing-because-of-no-such-key-or-host-key-verification-failed-what-do-i-do.md

```diff
@@ -24,13 +24,19 @@ To give containers within Runnable access to those dependency repositories, we j
 SSH keys come in a pair, private and public. To generate a new ssh key pair, open up a terminal session on a Linux or Mac machine you trust. If you already have an SSH key that you would like to use, [skip this step](#addSSHGitHub).
 
 1. Navigate to the ~/.ssh/ directory. (Or create it, if it doesn't exist)
-`$ mkdir -p ~/.ssh/ && cd ~/.ssh`
+  ```
```

Comment:
These code blocks need to…

1. have a newline above them.
2. be indented with four spaces.

This way, they'll appear in their own paragraphs (with their own margins) inside the list item.